src/styles/card.scss

```diff
@@ -0,0 +1,127 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  gap: 30px 48px;
+
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    max-width: 200px;
+    border: 1px solid #f3f3f3;
+    transition-duration: 300ms;
+
+    &_image {
+      display: block;
+      margin: 0 auto 40px;
+      width: 160px;
+      height: 134px;
+    }
+
+    &_name {
+      display: block;
```

Comment:
The 'display: block;' declaration is unnecessary for the '&__name' element since '<a>' tags are inline elements and will not take the full width of the container by default. You can remove this line unless there is a specific reason for it to be block level.